src/Build.UnitTests/BackEnd/BuildManager_Tests.cs

```diff
@@ -198,12 +198,8 @@ public void VerifyEnvironmentSavedBetweenCalls()
         /// The final number of nodes has to be less or equal the number of nodes already in 
         /// the system before this method was called.
         /// </summary>
-#if RUNTIME_TYPE_NETCORE
-        [Theory(Skip = "https://github.com/Microsoft/msbuild/issues/1975")]
-#elif MONO
-        [Theory(Skip = "https://github.com/Microsoft/msbuild/issues/1240")]
-#else
-        [Theory(Skip = "https://github.com/Microsoft/msbuild/issues/2057")]
+#if FEATURE_NODE_REUSE
+        [Theory]
```

Comment:
@AndyGerlicher So I didn't really do any new robustness improvements for this test but I hate to leave it off . . . thoughts? I could enable the test only in master so as not to torpedo potential future 15.7 changes?

Reply:
yup, saw the same problem on my Linux machine just now. Pulled that change out of here.